self-tests.yml

```diff
@@ -323,6 +323,50 @@ tasks:
           content_type: application/x-gzip
           permissions: public-read
           display_name: dist-test-${distro_id}.tar.gz
+  - name: smoke-test
+    commands:
+      - command: git.get_project
+        type: system
+        params:
+          directory: gopath/src/github.com/evergreen-ci/evergreen
+      - func: set-up-mongodb
+      - func: run-make
+        vars:
+          target: "build"
+      - command: shell.exec
+        params:
+          shell: bash
+          script: |
+            set -o xtrace
+            set -o errexit
+
+            cp mongodb/mongoimport gopath/src/github.com/evergreen-ci/evergreen/scripts/smoke_test
+            cd gopath/src/github.com/evergreen-ci/evergreen/scripts/smoke_test
+            for fn in *.json; do
+              collection=$(echo $fn | cut -f 1 -d '.')
+              ./mongoimport --db mci --collection $collection --file $fn
+            done
```

Comment:
why not populate via the migration tools, or from go itself?

Reply:
I like this because of the simplicity of having .json files in a directory somewhere, which can be edited by hand/modified.

Reply:
I think we have code (or could/should add code) to read JSON into the DB from go, which I think would help the time from "download dist" to "running demo evergreen"